Schemas/RP14A.xsd

```diff
@@ -209,15 +248,15 @@
 			<xsd:pattern value="[A-CEGHJ-PR-TW-Za-ceghj-pr-tw-z]{1}[A-CEGHJ-NPR-TW-Za-ceghj-npr-tw-z]{1}[0-9]{6}[A-DFMa-dfm]{1}"/>
 		</xsd:restriction>
 	</xsd:simpleType>
-  <xsd:simpleType name="CaseReferenceType">
+	<xsd:simpleType name="CaseReferenceType">
 		<xsd:restriction base="xsd:string">
-      <xsd:pattern value="CN[0-9]{8}|cn[0-9]{8}|Cn[0-9]{8}|cN[0-9]{8}"/>
+			<xsd:pattern value="CN[0-9]{8}|cn[0-9]{8}|Cn[0-9]{8}|cN[0-9]{8}"/>
 		</xsd:restriction>
 	</xsd:simpleType>
 	<xsd:simpleType name="YesNoType">
 		<xsd:restriction base="xsd:string">
-			<xsd:enumeration value="yes"/>
```

Comment:
This changes the content from yes/no to Y/N, is there a reason for this? If we're using that field type already then those will have to be changed, and even if we're not, then it might mean a change to client code that's included this type definition.

Reply:
After discussion I have reverted the yes/no type to as it was before